server/routes/getOffences.ts

```diff
@@ -38,7 +38,8 @@ const getOffences = async (models, req: Request, res: Response, next: NextFuncti
     ]
   });
   // No Offences
-  if (!offences.length) return [];
+  if (offences) { if (!offences.length) return []; } else { return []; }
```

Comment:
This is incorrect, you can't return a non-RESTFUL response

Reply:
Please just use: return `res.json({ status: 'Success', result: offences || [] });`

Reply:
Alright!